book/P3C13_classical.qmd

```diff
@@ -88,98 +88,145 @@ By first estimating $\hatF_X$ on separate training data, the estimator can be us
+<!--  -->
 $$
-\hat{H}_{Bres}(\tau) = \sum_{k:\tbk \leq \tau} \frac{d_{\tbk}}{\sum_{j \in \calR_{\tbk}} \exp(\hat{\eta}_j)}
+H_{Bres}(\tau) = H_0(\tau) = \sum_{k:\tbk \leq \tau} \frac{d_{\tbk}}{\sum_{j \in \calR_{\tbk}} \exp(\eta_j)}.
 $$ {#eq-breslow}
-
-where $\hat{\eta}_j$ is the estimated linear predictor for observation $j$.
+<!--  -->
 Note that if the value for all covariates or their effects was zero, or if there were no covariates, then the Breslow estimator is identical to the Nelson-Aalen estimator (@sec-surv-na):
 
 $$
-\hat{H}_{Bres}(\tau) = \sum_{k:\tbk \leq \tau} \frac{d_{\tbk}}{\sum_{j \in \calR_{\tbk}} 1} = \sum_{\tbk \leq \tau} \frac{d_{\tbk}}{n_{\tbk}} = \hat{H}_{NA}(\tau)
+H_{Bres}(\tau) = \sum_{k:\tbk \leq \tau} \frac{d_{\tbk}}{\sum_{j \in \calR_{\tbk}} 1} = \sum_{\tbk \leq \tau} \frac{d_{\tbk}}{n_{\tbk}} = H_{NA}(\tau).
 $$
 
-With these formulae, the Cox PH can be used as a predictive model by using training data to estimate $\hat{\bsbeta}$ with (@eq-partial).
+
+
+With these formulae, the Cox PH model can be used as a predictive model by using training data to estimate $\hat{\bsbeta}$ via (@eq-estimate-beta).
 These fitted coefficients are used to predict $\hat{\bseta}$ for new observations and finally the cumulative baseline hazard is computed with (@eq-breslow) to return a predicted distribution, for example the survival probability (@eq-ph-surv).
 
 <!-- FIXME -->
 The Cox model is is highly interpretable and has a long history of usage in clinical prediction modelling and analysis.
 However, the proportional hazards assumption is often violated in real life, leaving the model to be a questionable choice when used for data analysis or inference.
-One method to resolve this is to extend the model to handle time-varying coefficients and model stratification [@Cox1972], though it is surprisingly difficult to make meaningful and interpretable predictions from a time-varying model [@Reid1994].
-Whilst violation of the PH assumption can be a significant problem when interpreting model covariates, the goodness of fit is less important when using the model for predictive purposes.
-In fact, the Cox model often outperforms machine learning alternatives, including those that relax the PH assumption [@Burk2025, @Gensheimer2018; @Luxhoj1997; @VanBelle2011b].
+Nevertheless, years fo research have developed extensions of the cox model [@therneau2001modelingsurvival] that can incorporate stratified baseline hazards (the PH assumption only has to hold within strata), time-varying effects (the effects of time-constant covariates change over time) and time-varying covariates (the values of covariates change over time),
+though, especially in case of time-varying covariates, it is difficult to make meaningful and interpretable predictions (as the values of covariates might not be known at the time of prediction).
```

Comment:
```suggestion
However, especially in case of time-varying covariates, it is difficult to make meaningful and interpretable predictions (as the values of covariates might not be known at the time of prediction).
```